onnxruntime/core/providers/cpu/rnn/uni_directional_lstm.cc

```diff
@@ -376,6 +385,11 @@ void UniDirectionalLstm<T>::Compute(const gsl::span<const T>& inputs_arg,
           if (step >= min_sequence_length && step >= sequence_lengths[lrow]) {
             auto output_lrow = outputs.begin() + step * output_step_length + lrow * hidden_size_;
             std::fill_n(output_lrow, hidden_size_, (T)0);
+
+            if (training_mode_) {
+              auto all_cell_states_lrow = all_cell_states.begin() + step * output_step_length + lrow * hidden_size_;
+              std::fill_n(all_cell_states_lrow, hidden_size_, static_cast<T>(0));
```

Comment:
maybe we can align with line 387 "std::fill_n(output_lrow, hidden_size_, (T)0);"

Reply:
Do you mean use `(T)0` instead of `static_cast<T>(0)`?